src/commands/DriveToGateNoCompass.cpp

```diff
@@ -37,7 +37,7 @@ command_t DriveToGateNoCompass::getOutput()
 	{
 		calibrationPoints.clear();
 		double err = calculateHeadingErr();
-		double maxthvel = 1.0;
+		double maxthvel = 0.5;
```

Comment:
Just thought of this - it's not important now, but in the future this probably ought to be in `Constants.h` since it's used in so many places.

Reply:
```suggestion
		// TODO move to Constants.h
		double maxthvel = 0.5;
```